src/content/docs/Processes/Testing/overview.md

```diff
@@ -232,14 +232,17 @@ Please refer to the matrix below for device priorities:
 |--------------|---------------------------|----------|-----------|-----------|-----------|
 | Windows 11   | Windows Laptop            | P1 | P1 | P2 | -         |
 | MacOS        | Macbook                   | P1 | -         | P2 | P1|
-| iOS  | iPhone 15 series         | P1 | -         | -         | P1|
-| iOS             | iPhone 14 series          | P2 | -         | -         | P2|
-| iOS             | iPhone 13 and 12 series   | P2 | -         | -         | P1|
-| iPad OS (17) | iPhone SE (2nd)           | P1 | -         | -         | P1|
-| iPad OS (17)             | iPad Pro (10.5-inch) (5th generation) | P3 | -         | -         | P2|
-| iPad OS (17)             | iPad Air (5th generation) | P3 | -         | -         | P2|
-| Android 12   | Google Pixel 7            | P2 | -         | -         | -         |
-| Android 12        | Samsung Galaxy S24 and S24+| P1 | -         | -         | -         |
+| iOS (17,18) | iPhone 16 series         | P1 | -         | -         | P1|
```

Comment:
I don't think testing across all 12, 13, 14, 15 and 16 is as much valuable. Drop at least one or two of them. 

Maybe combine them. instead of 13 and 12, it becomes 13 or 12 series.
Combine 14 and 15 too, to say 14 or 15 series.

Reply:
I'd also suggest making the 13/12 series to be P3.